kikit/fab/neodenyy1.py

```diff
@@ -0,0 +1,164 @@
+import click
+from pcbnewTransition import pcbnew
+import csv
+import os
+import sys
+import shutil
+import re
+from pathlib import Path
+from kikit.fab.common import *
+from kikit.common import *
+from kikit.export import gerberImpl
+mm_ius = 1000000.0
```

Comment:
We already have a constant for this in the units module

Reply:
Fixed, now using units module.